2022-12-02 11:26:29,275 INFO    MainThread:1543018 [internal.py:wandb_internal():90] W&B internal server running at pid: 1543018, started at: 2022-12-02 11:26:29.272310
2022-12-02 11:26:29,309 INFO    WriterThread:1543018 [datastore.py:open_for_write():75] open: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20221202_112627-3oj8ggf5/run-3oj8ggf5.wandb
2022-12-02 11:26:29,310 DEBUG   HandlerThread:1543018 [handler.py:handle_request():141] handle_request: run_start
2022-12-02 11:26:29,329 DEBUG   HandlerThread:1543018 [meta.py:__init__():35] meta init
2022-12-02 11:26:29,329 DEBUG   HandlerThread:1543018 [meta.py:__init__():49] meta init done
2022-12-02 11:26:29,329 DEBUG   HandlerThread:1543018 [meta.py:probe():209] probe
2022-12-02 11:26:29,339 DEBUG   HandlerThread:1543018 [meta.py:_setup_git():199] setup git
2022-12-02 11:26:29,467 DEBUG   HandlerThread:1543018 [meta.py:_setup_git():206] setup git done
2022-12-02 11:26:29,468 DEBUG   HandlerThread:1543018 [meta.py:_save_pip():53] save pip
2022-12-02 11:26:29,471 DEBUG   HandlerThread:1543018 [meta.py:_save_pip():67] save pip done
2022-12-02 11:26:29,472 DEBUG   HandlerThread:1543018 [meta.py:_save_conda():74] save conda
2022-12-02 11:26:48,195 DEBUG   HandlerThread:1543018 [meta.py:_save_conda():84] save conda done
2022-12-02 11:26:48,195 DEBUG   HandlerThread:1543018 [meta.py:probe():247] probe done
2022-12-02 11:27:28,404 WARNING MainThread:1543018 [internal.py:wandb_internal():153] Internal process interrupt: 1
2022-12-02 11:27:28,855 WARNING MainThread:1543018 [internal.py:wandb_internal():153] Internal process interrupt: 2
2022-12-02 11:27:28,855 ERROR   MainThread:1543018 [internal.py:wandb_internal():156] Internal process interrupted.
2022-12-02 11:27:29,305 INFO    SenderThread:1543018 [sender.py:finish():1111] shutting down sender
2022-12-02 11:27:29,709 INFO    WriterThread:1543018 [datastore.py:close():279] close: /fsx/ganayu/code/SuperShaper/wandb/offline-run-20221202_112627-3oj8ggf5/run-3oj8ggf5.wandb
2022-12-02 11:27:29,724 INFO    HandlerThread:1543018 [handler.py:finish():806] shutting down handler
2022-12-02 11:27:29,726 INFO    MainThread:1543018 [internal.py:handle_exit():80] Internal process exited
